pkgs/applications/networking/netclient/default.nix

```
@@ -0,0 +1,42 @@
+{ buildGoModule
+, fetchFromGitHub
+, installShellFiles
+, lib
+, libglvnd
+, pkg-config
+, xorg
+}:
+
+buildGoModule rec {
+  pname = "netclient";
+  version = "0.20.3";
+
+  src = fetchFromGitHub {
+    owner = "gravitl";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-GeXmAwmGbb73Z0OhW7cNu6UkccpaHqVQOItEK58Jh1o=";
+  };
+
+  vendorHash = "sha256-qchVMLmdvqpVpyjDy05W8yHTBcYCgpyuX5Ibg+yceXY=";
+
+  buildInputs = [
+    libglvnd
+    xorg.libX11
+    xorg.libXcursor
+    xorg.libXi
+    xorg.libXinerama
+    xorg.libXrandr
+  ];
+
+  postInstall = ''
+    install -Dm444 build/netclient.service -t $out/lib/systemd/netclient.service
+  '';
+
+  meta = with lib; {
+    description = "Automated WireGuard® Management Client";
+    homepage = "https://netmaker.io";
+    changelog = "https://github.com/gravitl/netclient/-/releases/v${version}";
+    license = licenses.sspl;
```

wrong license, they use Apache

```suggestion
    license = licenses.asl20;
```